crates/floresta-chain/src/pruned_utreexo/consensus.rs

```diff
@@ -233,13 +346,28 @@ impl Consensus {
         };
         Ok(())
     }
-    #[allow(unused)]
     fn validate_locktime(
         input: &TxIn,
         transaction: &Transaction,
         height: u32,
+        mtp: u32,
     ) -> Result<(), BlockValidationErrors> {
-        unimplemented!("validate_locktime")
+        if input.sequence.is_height_locked() || input.sequence.enables_absolute_lock_time() {
+            if !transaction.is_absolute_timelock_satisfied(
+                Height::from_consensus(height).unwrap(),
+                Time::from_consensus(mtp).unwrap(),
+            ) {
+                return Err(BlockValidationErrors::BadAbsoluteLockTime);
+            }
+        }
+        if input.sequence.is_relative_lock_time() {
+            //validate lock time
+            //TODO needs to find te block that the outpoint was created
+        }
+        if input.sequence.is_rbf() {
```

Comment:
RBF is a mempool policy, we don't have mempool. I don't think this is needed

Reply:
yes, thats why the if is empty... Ill just delete it, was for notation